src/controllers/bookmarkController.js

```diff
@@ -0,0 +1,105 @@
+import { Bookmark, User, Article } from '../sequelize/models';
+
+/**
+ * @description holds bookmarks logic
+ */
+class BookmarkController {
+  /**
+   * @description create bookmarks
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} returns object with created bookmark
+   */
+  static async createBookmark(req, res) {
+    try {
+      const userId = req.userData.id;
+      const articleSlug = req.params.slug;
+      const user = await User.findOne({ where: { id: userId } });
```

Comment:
This line is not needed if this routes is protected.

Reply:
We need that line,  imagine a scenario where a user is deleted from the system but the token is still a valid one and when one tries to use it will end up throwing an error. This is because the verifyToken middleware is only used to decode the token but not check the user if exist.

Reply:
Yes I get you. So can you please make it as a middleware so that can be reused in future by another team mate.